<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TempoMarkerDialog</class>
 <widget class="QDialog" name="TempoMarkerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tempo Marker</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="descriptionComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="beatTypeLabel">
       <property name="text">
        <string>Beat Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="note2Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/half_note</normaloff>:/images/half_note</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="dottedNote2Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/half_note_dotted</normaloff>:/images/half_note_dotted</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="note4Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/quarter_note</normaloff>:/images/quarter_note</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="dottedNote4Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/dotted_note</normaloff>:/images/dotted_note</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="note8Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/8th_note</normaloff>:/images/8th_note</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="dottedNote8Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/8th_note_dotted</normaloff>:/images/8th_note_dotted</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="note16Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/16th_note</normaloff>:/images/16th_note</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="dottedNote16Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/16th_note_dotted</normaloff>:/images/16th_note_dotted</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="note32Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/32nd_note</normaloff>:/images/32nd_note</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="dottedNote32Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/32nd_note_dotted</normaloff>:/images/32nd_note_dotted</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="bpmLabel">
       <property name="text">
        <string>BPM:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="bpmSpinBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lIstessoLabel">
       <property name="text">
        <string>L'istesso:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="listessoNote2Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/half_note</normaloff>:/images/half_note</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="listessoDottedNote2Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/half_note_dotted</normaloff>:/images/half_note_dotted</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="listessoNote4Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/quarter_note</normaloff>:/images/quarter_note</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="listessoDottedNote4Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/dotted_note</normaloff>:/images/dotted_note</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="listessoNote8Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/8th_note</normaloff>:/images/8th_note</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="listessoDottedNote8Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/8th_note_dotted</normaloff>:/images/8th_note_dotted</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="listessoNote16Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/16th_note</normaloff>:/images/16th_note</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="listessoDottedNote16Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/16th_note_dotted</normaloff>:/images/16th_note_dotted</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="listessoNote32Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/32nd_note</normaloff>:/images/32nd_note</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="listessoDottedNote32Button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/32nd_note_dotted</normaloff>:/images/32nd_note_dotted</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../build/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TempoMarkerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TempoMarkerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
